packages/main/test/pages/BusyIndicator.html

```diff
@@ -70,6 +70,34 @@
 		<ui5-list id="fetch-list" no-data-text="No Data" header-text="Available Items" class="busyindicator5auto"></ui5-list>
 	</ui5-busy-indicator>
 
+	<br>
+	<br>
+	<br>
+	<br>
+
+	<ui5-table loading id="tableloading" overflow-mode="Popin">
+		<ui5-table-header-row slot="headerRow">
+			<ui5-table-header-cell id="produtCol" width="300px"><span>Product</span></ui5-table-header-cell>
+			<ui5-table-header-cell id="supplierCol" width="200px">Supplier</ui5-table-header-cell>
+			<ui5-table-header-cell id="dimensionsCol" width="300px">Dimensions</ui5-table-header-cell>
+			<ui5-table-header-cell id="weightCol" width="100px">Weight</ui5-table-header-cell>
+			<ui5-table-header-cell id="priceCol" width="220px">Price</ui5-table-header-cell>
+		</ui5-table-header-row>
+		<ui5-table-row>
+			<ui5-table-cell><ui5-label><b>Notebook Basic 15</b><br>HT-1000</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label>Very Best Screens</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label>30 x 18 x 3 cm</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label style="color: #2b7c2b"><b>4.2</b> KG</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label><b>956</b> EUR</ui5-label></ui5-table-cell>
+		</ui5-table-row>
+		<ui5-table-row>
+			<ui5-table-cell><ui5-label><b>Notebook Basic 17</b><br>HT-1001</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label>Smartcards</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label>29 x 17 x 3.1 cm</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label style="color: #2b7c2b"><b>4.5</b> KG</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label><b>1249</b> EUR</ui5-label></ui5-table-cell>
+		</ui5-table-row>
+	</ui5-table>
```

Comment:
Please remove this to keep the test page clean of other components' specific usages. Table is responsible for its own loading behavior